testing/firefox/fix-rust-target.patch

```diff
@@ -0,0 +1,11 @@
+--- a/build/moz.configure/rust.configure	2017-05-27 10:48:23.927528005 +0000
++++ b/build/moz.configure/rust.configure	2017-05-27 10:52:42.973790729 +0000
+@@ -130,7 +130,7 @@
+             ('arm', 'Linux'): 'armv7-unknown-linux-gnueabihf',
+             ('sparc64', 'Linux'): 'sparc64-unknown-linux-gnu',
+             ('x86', 'Linux'): 'i686-unknown-linux-gnu',
+-            ('x86_64', 'Linux'): 'x86_64-unknown-linux-gnu',
```

Comment:
iight. I'm going to work on enabling Rust on other architectures - Void's thing is not an issue as we can cross-bootstrap arches now that we have our own binaries, it's just used out of convenience as it works with our build servers until we have our own bootstrap.